webhook_server/tests/test_github_api.py

```
@@ -876,13 +876,27 @@ async def test_process_status_event(self, state: str, should_recheck: bool) -> N
-                                    else:
-                                        mock_pr_handler.return_value.check_if_can_be_merged.assert_not_awaited()
+                                    with patch.object(
+                                        GithubWebhook,
+                                        "_clone_repository",
+                                        new=AsyncMock(return_value=None),
+                                    ) as mock_clone:
+                                        with patch.object(
+                                            OwnersFileHandler,
+                                            "initialize",
+                                            new=AsyncMock(return_value=None),
+                                        ) as mock_owners_init:
```

Ruff (0.15.6)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)